examples/tuono-api/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "tuono-api"
+version = "0.0.1"
+edition = "2021"
+
+[[bin]]
+name = "tuono"
+path = ".tuono/main.rs"
+
+[dependencies]
+tuono_lib = { path = "../../crates/tuono_lib/", features = ["api"]}
```

Comment:
In the description you speak about `api-only` mode, 
however the feature flag is just named `api`.
Furthermore, always in the PR description, is stated:

> By just adding the api feature flag to tuono_lib the CLI automatically switches to the api-only mode, entirely stripping out the SSR from the project.

So, how about using `api-only` also as feature flag name?